config/config.go

```diff
@@ -158,15 +171,59 @@ const (
 	MetricsInflux MetricsType = "influx"
 )
 
-type Influx struct {
+type InfluxMetrics struct {
 	Host     string `mapstructure:"host"`
 	Database string `mapstructure:"database"`
 	Username string `mapstructure:"username"`
 	Password string `mapstructure:"password"`
+	Enabled  bool   `mapstructure:"enabled"`
+}
+
+func (influxMetricsConfig *InfluxMetrics) validateAndLogMetricsData() bool {
```

Comment:
Consider renaming to something like `validateAndLog`. It's already associated with the metrics config object.

Reply:
Done